File: SarfMalzemeStok.Domain/Model/HistoryInformationReport.cs
using Abp.Domain.Entities;
using System;
using System.Collections.Generic;
using System.Text;

namespace SarfMalzemeStok.Domain.Model
{
    public class HistoryInformationReport: Entity
    {
        public int MaterialId { get; set; }
        public string Process{ get; set; }
        public string PartNumber{ get; set; }
        public string DescPart{ get; set; }
        public string Operation{ get; set; }
        public string WorkStation{ get; set; }
        public string Consumable{ get; set; }
        public string OlcuBirimi{ get; set; }
        public string UPP{ get; set; }
        public string UPPDegisiklik{ get; set; }
        public string DegisiklikYapan{ get; set; }

        public Material material { get; set; }
    }
}

File: SarfMalzemeStok/ClientApp/src/app/models/historyInformationReport.ts
import { Material } from './material';

export class HistoryInformationReport {
    id:number;
    materialId:number;
    process:string;
    partNumber:string;
    descPart:string;
    operation:string;
    workStation:string;
    consumable:string;
    olcuBirimi:string;
    upp:string;
    uppDegisiklik:string;
    degisiklikYapan:string;

    material:Material;
  
}
